Documentation/ApiOverview/Routing/AdvancedRoutingConfiguration.rst

```diff
@@ -181,6 +181,10 @@ The configuration looks like this:
           category_id: 'category'
 
 :yaml:`routePath` defines the static keyword and the placeholders.
+:yaml:`requirements.category_id` is a unique integer of the category which can be the uid in its database table.
```

Comment:
The example here only shows that there is an argument in URL called "category" `&category=003`. With `_arguments` a mapping to a "better" name of that argument name happens and the new name `category_id` will be checked against a reg exp. Nothing else happens here. No table lookup, no UID, no record. The number can also be the second or third category of whatever.
As long as no aspect with more details on how to "validate" that category value, it is just a 1 to 1 mapping from URL argument to a slug.